samples/wifi/shutdown/Kconfig

```
@@ -13,3 +13,34 @@ config WIFI_MAC_ADDRESS
 		WiFi MAC address to be used by the sample. If not set, the sample will
 		use the default MAC address (locally administrative). This is only used
 		when the nRF7002 OTP is not programmed with a MAC address.
+
+choice SHUTDOWN_OPERATION_MODE
+	prompt "Sample operation mode"
+	default OPERATION_MODE_CONTINUOUS
+	help
+		Select the operation mode for the sample.
+
+config OPERATION_MODE_CONTINUOUS
+	bool "Continuous mode"
+	help
+		In this mode, the sample will continue to run alternating between
+		WiFi shutdown and startup.
+
+config OPERATION_MODE_ONE_SHOT
+	bool "One-shot mode"
+	help
+		In this mode, the sample will shutdown WiFi only once and then exit.
+
+config OPERATION_MODE_BUTTONS
+	bool "Enable button interaction"
+	help
+		Enable button interaction in the sample. If enabled, the sample will
+		enable use of buttons to toggle the WiFi shutdown or startup.
+endchoice
+
+config SHUTDOWN_TIMEOUT
```

```suggestion
config SHUTDOWN_TIMEOUT_S/MS
```
I guess seconds is reasonable than ms?

---

Yeah, seconds is better.